File: project_euler/00/006.go
// Copyright [2017] <dmnsn7@gmail.com>

package main

import "fmt"

const n = 100

func main() {
	sum1 := n * (n + 1) / 2
	sum2 := n * (n + 1) * (2*n + 1) / 6
	ans := sum1*sum1 - sum2

	fmt.Printf("%d\n", ans)
}

File: project_euler/00/006.py
#!/usr/bin/env python3
'''Copyright [2017] <dmnsn7@gmail.com>'''

N = 100


def main():
    sum_1 = N * (N + 1) // 2
    sum_2 = N * (N + 1) * (2 * N + 1) // 6
    ans = sum_1 * sum_1 - sum_2

    print(ans)


if __name__ == '__main__':
    main()
